torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1,14 +1,84 @@
+#include <stack>
 #include <torch/csrc/jit/passes/quantization.h>
 
 #include <torch/csrc/jit/ir.h>
 #include <torch/csrc/jit/node_hashing.h>
 #include <torch/csrc/jit/passes/alias_analysis.h>
 
-#include <stack>
+#include <unordered_set>
 
 namespace torch {
 namespace jit {
+namespace {
+// QuantizerUtils
+
+int checkIfNodeValuesQuantizable(Node* n) {
```

Comment:
This can be `bool` instead of `int`. Also, it could use a better name: when one reads `if(checkSomething)` it's less obvious what the check returns compared to `if(isSomething)` for instance.